src/sentry/models/projectownership.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from jsonfield import JSONField
+
+from django.db import models
+from django.utils import timezone
+from sentry.db.models import Model, sane_repr
+from sentry.db.models.fields import FlexibleForeignKey
+
+
+class ProjectOwnership(Model):
+    __core__ = True
+
+    project = FlexibleForeignKey('sentry.Project', unique=True)
+    raw = models.TextField(null=True)
+    schema = JSONField(null=True)
+    fallthrough = models.BooleanField(default=True)
+    date_created = models.DateTimeField(default=timezone.now)
+    last_updated = models.DateTimeField(default=timezone.now)
+    is_active = models.BooleanField(default=True)
```

Comment:
There's currently no UX or demand to enable/disable, but just thinking ahead if we wanted this.